packages/grafana-schema/src/schema/pluginmeta/pluginmeta_experimental.gen.ts

```diff
@@ -0,0 +1,51 @@
+// This file is autogenerated. DO NOT EDIT.
+//
+// Run "make gen-cue" from repository root to regenerate.
+//
+// Derived from the Thema lineage at pkg/coremodel/pluginmeta
+
+
+// This model is a WIP and not yet canonical. Consequently, its members are
+// not exported to exclude it from grafana-schema's public API surface.
+
+interface Pluginmeta {
+  alerting?: boolean;
+  annotations?: boolean;
+  autoEnabled?: boolean;
+  backend?: boolean;
+  category?: ('tsdb' | 'logging' | 'cloud' | 'tracing' | 'sql' | 'enterprise' | 'other');
+  dependencies: {
+    grafanaDependency: string;
+  };
+  enterpriseFeatures?: {};
+  executable?: string;
+  hiddenQueries?: boolean;
+  id: string;
+  includes?: {}[];
```

Comment:
Seems like this might also be a bug in cuetsy - nested optional structs don't get their fields generated

Reply:
Yup, it's another bug to squash. grafana/cuetsy#58